examples/mask.py

```diff
@@ -36,92 +37,87 @@
 one or more image file names as command line arguments.
 """
 
-import sys
 import os
 import random
+import sys
 
 import pygame as pg
 
 
-def maskFromSurface(surface, threshold=127):
-    return pg.mask.from_surface(surface, threshold)
-
-
-def vadd(x, y):
-    return [x[0] + y[0], x[1] + y[1]]
-
-
-def vsub(x, y):
-    return [x[0] - y[0], x[1] - y[1]]
-
-
-def vdot(x, y):
-    return x[0] * y[0] + x[1] * y[1]
-
-
 class Sprite:
-    def __init__(self, surface, mask=None):
+    """
+    Moving Sprite demonstrating pixel-perfect collisions between pg.mask.Mask objects
+    """
+
+    def __init__(self, pos, vel, surface: pg.Surface, mask: pg.mask.Mask = None):
```

Comment:
I think this is inconsistent typehinting because some args are typed and some are not, it should be all or none IMHO (I'm personally leaning on the latter for now because most of our other examples don't have type hints, but I'll let you take the call)

If you do decide to go with the typehinting, there is a small issue here: The `mask` argument should be typed like `Optional[pg.mask.Mask]`

Reply:
Thanks for checking out my PR! 🥳 
Decided to remove the typehints